package.json

```diff
@@ -22,7 +22,7 @@
     "lint": "node ./bin/balena-lint -t tsconfig.json lib/ config/eslint.config.js",
     "prepublish": "require-npm4-to-publish",
     "prepublishOnly": "npm run test",
-    "lint-fix": "node ./bin/balena-lint -t tsconfig.json --fix -e ts -e js -t tsconfig.json lib/ config/eslint.config.js",
+    "lint-fix": "node ./bin/balena-lint -t tsconfig.json --fix -e ts -e js lib/ config/eslint.config.js",
```

Comment:
Unrelated, but there shouldn't be a good reason to specify `-t tsconfig.json` twice